collector/default_collector.go

```diff
@@ -0,0 +1,76 @@
+package collector
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const NAMESPACE = "sap"
```

Comment:
I was thinking if perhaps sap or `sap_host` . we can find out later

Reply:
Bear in mind that we will probably have a bunch of subsystems to group the metrics in, like `ascs`  and `ers`, this is why I left "host" out.

Reply:
This is minor. I think we can resolve it later. No need right now